h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -61,8 +62,8 @@
   
   @Before
   public void setupMojoJavaScoring() {
-    System.setProperty("sys.ai.h2o.xgboost.scoring.java.enable", confMojoJavaScoring); // mojo scoring
-    System.setProperty("sys.ai.h2o.xgboost.predict.java.enable", confJavaPredict); // in-h2o predict
+    System.setProperty(XGBoostMojoReader.SCORE_JAVA_PROP, confMojoJavaScoring); // mojo scoring
```

Comment:
I usually avoid doing this, it is better to have the constants hard-coded in tests. An unintended edit in the constant will break functionality for users but tests will stay green.